Integration jobs fails on prometheus alerts about Kubernetes API certificate expiration

**Description**
Such errors are reported on [kyma-alpha-integration-evaluation-gardener-azure](https://status.build.kyma-project.io/job-history/gs/kyma-prow-logs/logs/kyma-alpha-integration-evaluation-gardener-azure) and [kyma-alpha-integration-production-gardener-azure](https://status.build.kyma-project.io/job-history/gs/kyma-prow-logs/logs/kyma-alpha-integration-production-gardener-azure) jobs:
```
  1) Executing Standard Testsuite:
       Prometheus Tests:
         No critical Prometheus alerts should be firing:
     AssertionError: Following alerts are firing: {
  labels: { alertname: [32m'K8sCertificateExpirationNotice2'[39m, severity: [32m'warning'[39m },
  annotations: {
    description: [32m'Kubernetes API Certificate is expiring soon (less than 7 days)'[39m
  },
  state: [32m'firing'[39m,
  activeAt: [32m'2022-02-02T13:18:14.117849851Z'[39m,
  value: [32m'4.962e+03'[39m
}: expected [ { labels: 
     { alertname: 'K8sCertificateExpirationNotice2',
       severity: 'warning' },
    annotations: { description: 'Kubernetes API Certificate is expiring soon (less than 7 days)' },
    state: 'firing',
    activeAt: '2022-02-02T13:18:14.117849851Z',
    value: '4.962e+03' } ] to be empty
      at Object.assertNoCriticalAlertsExist (monitoring/prometheus.js:49:10)
      at async Context.<anonymous> (test/3-monitoring.js:30:7)
```

--- a/tests/fast-integration/monitoring/prometheus.js
+++ b/tests/fast-integration/monitoring/prometheus.js
@@ -156,12 +156,14 @@ function shouldIgnoreAlert(alert) {
     'Watchdog',
     // Scrape limits can be exceeded on long-running clusters and can be ignored
     'ScrapeLimitForTargetExceeded',
-    // Overcommitting resources is fine for e2e test scenarios
+    // Resource overcommitment is fine for e2e test scenarios
     'KubeCPUOvercommit',
     'KubeMemoryOvercommit',
+    // API server certificates are auto-renewed
+    'K8sCertificateExpirationNotice'
   ];
 
-  return alert.labels.severity == 'critical' || alertNamesToIgnore.includes(alert.labels.alertname);
+  return alert.labels.severity != 'critical' || alertNamesToIgnore.includes(alert.labels.alertname);
 }
 
 async function getServiceMonitors() {

--- a/tests/fast-integration/monitoring/prometheus.js
+++ b/tests/fast-integration/monitoring/prometheus.js
@@ -160,7 +160,7 @@ function shouldIgnoreAlert(alert) {
     'KubeCPUOvercommit',
     'KubeMemoryOvercommit',
     // API server certificates are auto-renewed
-    'K8sCertificateExpirationNotice'
+    'K8sCertificateExpirationNotice',
   ];
 
   return alert.labels.severity != 'critical' || alertNamesToIgnore.includes(alert.labels.alertname);

